Scripts/script-FailedInstances.yml

```diff
@@ -0,0 +1,72 @@
+commonfields:
+  id: FailedInstances
+  version: -1
+fromversion: 4.0.0
```

Comment:
is there specific reason why this script is `fromversion: 4.0.0`?

Reply:
@anara123 , yes, it uses the test module feature which is only supported in v4.